web-www/pages/contact.tsx

```diff
@@ -219,12 +223,24 @@ export default function ContactPage({
                           send
                         </ContainedButton>
                       </Form>
-                      {submitCount > 0 && !isSubmitting && (
-                        <Banner
-                          duration={bannerDuration}
-                          text="Your message was sent to the Regen team!"
-                        />
-                      )}
+                      {submitCount > 0 &&
+                        !isSubmitting &&
```

Comment:
We could extract that into a `hasBeenSubmitted` variable since it's reused twice.